src/pybind/mgr/orchestrator/module.py

```diff
@@ -1188,6 +1188,27 @@ def _apply_rgw(self,
                 out = to_format(data, format, many=True, cls=None)
         return HandleCommandResult(stdout=out)
 
+    @_cli_write_command(
+        'orch apply ha_rgw',
+        'Update the number of ha_rgw instances')
```

Comment:
The help string is not correct. Maybe something like "Create an High Availability RGW service bases in the existing RGW daemons"